.github/workflows/publish.yaml

```diff
@@ -103,23 +105,15 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
 
-      - name: Build wheels - x86_64
-        uses: messense/maturin-action@683eb33e2cda131da55757dbac7628b2ef22df2f
+      - name: Build wheels - ${{ matrix.target }}
+        uses: PyO3/maturin-action@683eb33e2cda131da55757dbac7628b2ef22df2f
         env:
           PYO3_PYTHON: python${{ matrix.python-version }}
         with:
-          target: x86_64-apple-darwin
+          target: ${{ matrix.target }}
           command: build
           args: --release -o dist
 
-      - name: Build wheels - universal2
-        uses: messense/maturin-action@683eb33e2cda131da55757dbac7628b2ef22df2f
-        env:
-          PYO3_PYTHON: python${{ matrix.python-version }}
-        with:
-          command: build
-          args: --release -o dist --universal2
```

Comment:
Just to make sure, the `--universal2` flag is not necessary when `target` is passed to the action?